src/tests/UnitTests/Navigation/WPF/NavigationServiceTests.show.cs

```diff
@@ -0,0 +1,162 @@
+﻿using CodeMonkeys.UnitTests.Navigation.ViewModels;
+
+using NUnit.Framework;
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace CodeMonkeys.UnitTests.Navigation.WPF
+{
+    public partial class NavigationServiceTests
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5575746)